docs/tools/vdb_table/data/vespa.json

```diff
@@ -98,7 +98,7 @@
     "comment": ""
   },
   "llamaindex": {
-    "support": "none",
+    "support": "full",
     "source_url": "",
```

Comment:
Can you add a link to documentation/starter notebook for doing the integration here?
Getting the same added to https://docs.llamaindex.ai/en/stable/module_guides/storing/vector_stores/ would also be great. Thanks

Reply:
Yes, so the starter notebook is not published to the docs yet, as the integration and main llama-index repo have different release cycle. 
The integration was added in this PR: 
- https://github.com/run-llama/llama_index/pull/13213

Here is the link to the starter notebook on Github (that will be part of docs on next release of main repo):
- https://github.com/run-llama/llama_index/blob/7a678073db9c0a602448cd75c99c20a50b28aad6/docs/docs/examples/vector_stores/VespaIndexDemo.ipynb#L4

Thanks for the pointer to the module guide, hadn`t seen that one 😅 .
 Made a PR adding Vespa there as well:
- https://github.com/run-llama/llama_index/pull/13326

Reply:
Is that good enough @dhruv-anand-aintech? :)

Reply:
Now, the notebook is published to the docs, so added the link there.